js/kucoinfutures.js

```diff
@@ -1089,7 +1115,7 @@ module.exports = class kucoinfutures extends kucoin {
                 throw new ArgumentsRequired (this.id + ' createOrder() requires a visibleSize parameter for iceberg orders');
             }
         }
-        params = this.omit (params, [ 'timeInForce', 'stopPrice', 'triggerPrice' ]); // Time in force only valid for limit orders, exchange error when gtc for market orders
+        params = this.omit (params, [ 'timeInForce', 'stop', 'stopPrice', 'stopPriceType', 'triggerPrice', 'stopLossPrice', 'takeProfitPrice' ]); // Time in force only valid for limit orders, exchange error when gtc for market orders
```

Comment:
Exchange specific params like `stop` and `stopPriceType` shouldn't be removed, if they are set in params, then the value will be overwritten by `extend`. Take a look at https://github.com/ccxt/ccxt/pull/16619 , it sets a default value of `'MP'` and if the user passes `stopPriceType` to params, then the users value takes priority